spec/features/recording_maneuver_score_spec.rb

```diff
@@ -21,3 +21,46 @@
     expect(page).to have_text "recorded by #{judge.name}"
   end
 end
+
+describe 'updating a maneuver score' do
+  let!(:maneuver_participant) do
+    create :maneuver_participant, :perfect_score
+  end
+  it 'updates the value' do
+    when_current_user_is :admin
+    visit maneuver_participant_path(maneuver_participant.id)
+    fill_in 'reversed_direction', with: '1'
+    click_on 'Save score'
+    input = find_field :reversed_direction
+    expect(input.value).to eql '1'
+  end
+end
+
+describe 'parsing from string to integer' do
```

Comment:
That's more of a description of the internal workings of the code than an actual feature.